search-app/src/cmr/search/services/query_service.clj

```diff
@@ -272,11 +272,13 @@
   (when-let [[start-date end-date] (mapv time-format/parse
                                          (string/split (:has-granules-created-at params) #","))]
     (let [query (qm/query {:concept-type :granule
-                           :condition (qm/date-range-condition
-                                       :created-at start-date end-date)
-                           :page-size :unlimited
+                           :page-size 0
```

Comment:
I don't know enough about how the code is used to anticipate the (possibly non-obvious) impacts of changing the page-size.